### PR TITLE
Np 5629/stop serving server only services

### PR DESCRIPTION
**Pull request by adamfox-nanopay**

### CODE REVIEW

File: src/foam/nanos/session/services [hunk 1]
```diff
@@ -36,7 +36,7 @@ p({
 p({
   "class": "foam.nanos.boot.NSpec",
   "name": "sessionService",
-  "serve": true,
+  "serve": false,
```
Comment (jlhughes) [inline src/foam/nanos/session/services]: The app works with this?  The whole point of the sessionService is for the client.  This suggests we don't need this service.

File: src/foam/nanos/theme/services.jrl [hunk 2]
```diff
@@ -43,7 +43,7 @@ p({
 p({
   "class": "foam.nanos.boot.NSpec",
   "name": "themes",
-  "serve": true,
+  "serve": false,
```
Comment (jlhughes) [inline src/foam/nanos/theme/services.jrl:46]: This suggest, this entire nspec is not needed.

File: src/services [hunk 3]
```diff
@@ -261,7 +261,7 @@ p({
 p({
   "class":"foam.nanos.boot.NSpec",
   "name":"countryService",
-  "serve":true,
+  "serve":false,
```
Comment (jlhughes) [inline src/services:264]: thinking for many, that the entire nspec is not needed.

File: src/foam/nanos/theme/services.jrl (hunk 2 above)
Comment (kgrgreer) [reply src/foam/nanos/theme/services.jrl:46]: It is used in several places: 
https://github.com/kgrgreer/foam3/search?q=.get%28%22themes%22%29

File: src/services (hunk 3 above)
Comment (kgrgreer) [reply src/services:264]: Used here:
https://github.com/nanoPayinc/NANOPAY/search?q=get%28%22countryService%22%29

**Pull request merged**